provider/resource_entity_type.go

```diff
@@ -172,29 +182,35 @@ func (r *resourceEntityType) dashboardDrilldownSchema() schema.SetNestedBlock {
 						This setting exists because for internal purposes, navigation suggestions are treated as dashboards.
 						This setting is only required if is_splunk_dashboard is false.
 					`),
-					Optional: true,
-					Computed: true,
-					Default:  stringdefault.StaticString(""),
+					Required: true,
+					// Optional: true,
+					// Computed: true,
+					// Default:  stringdefault.StaticString(""),
```

Comment:
> where the planned value may be wrong/different from what's specified in the terraform config.
it was that the default value overrides what in config?

Reply:
i remember that `dashboard_id` was getting set to null (while having legit string values set in the config), and i've seen a case when `dashboard_type` was getting set to (at plan time) to `navigation_link`, while it was actually `xml_dashboard` in the config 🤯